massdriver.yaml

```
@@ -2,7 +2,7 @@ schema: draft-07
 name: azure-aks-cluster
 description: "Azure Kubernetes Service (AKS) is a fully managed container orchestration service. AKS offers serverless Kubernetes, an integrated continuous integration and continuous delivery (CI/CD) experience, and enterprise-grade security and governance."
 source_url: github.com/massdriver-cloud/azure-aks-cluster
-access: "public"
+access: private
```

```suggestion
access: public
```
Whenever this is ready to merge, we should swap this back 👍